jena/src/main/resources/META-INF/resources/index.xed

```diff
@@ -1,40 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<webpage lastModified="$Date$" id="dozbib">
- <title xml:lang="de">Willkommen bei der Universitätsbibliographie der FSU Jena</title>
- <title xml:lang="en">Welcome to Bibliography of FSU Jena</title>
+<webpage lastModified="2022-05-23T10:46:00" id="thunibib-jena-index">
+  <title xml:lang="de">Willkommen bei der Universitätsbibliographie Jena</title>
+  <title xml:lang="en">Welcome to the bibliography of Friedrich Schiller University Jena</title>
 
- <article class="card mb-3" xml:lang="de">
-   <div class="card-body">
-     <h3>Die Universitätsbibliographie</h3>
-     <p>
-       Die Universitätsbibliographie dokumentiert die Publikationstätigkeit der Angehörigen der
-       <a href="https://www.uni-jena.de/">Friedrich-Schiller-Universität Jena</a> (FSU) und wird von der
-       <a href="https://www.thulb.uni-jena.de/">Thüringer Universitäts- und Landesbibliothek Jena</a> betrieben.
-     </p>
-     <p>
-       Als Angehörige der FSU Jena können Sie <a href="../newPublication.xed">Ihre Publikationen an die
-       Universitätsbibliographie melden</a>.
-       Damit helfen Sie uns bei ihrer Vervollständigung.
-     </p>
-     <p>
-       Der Aufbau der Universitätsbibliographie wird durch das Land Thüringen im Rahmen der Thüringer Digitalstrategie
-       gefördert und erfolgt in Zusammenarbeit mit der Verbundzentrale des Gemeinsamen Bibliotheksverbunds.
-     </p>
-   </div>
- </article>
+  <article class="card mb-3" xml:lang="de">
+    <div class="card-body">
+      <h3>Die Universitätsbibliographie</h3>
```

Comment:
we could prevent redundant code usage, see https://github.com/ub-duisburg-essen/ubo-due/blob/new-texts-improved-rebased/src/main/resources/META-INF/resources/index.xed

Reply:
Fixed